app/src/main/res/values/dimes.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+  <dimen name="item_width">200dp</dimen>
+  <dimen name="app_bar_height">200dp</dimen>
```

Comment:
here's an interesting one, is it possible to do math with the tokens? like spacingXXL is 40dp can we do spacingXXL * 5?

Reply:
Unfortunately no. https://stackoverflow.com/questions/8101178/how-do-i-apply-mathematical-operations-to-android-dimensions

Also, the ones here are from generated for the demo app. In actual components, the dimensions would be set in kotlin files and would follow the suggestion.

Reply:
worth a shot 😄